cmd/vm.go

```diff
@@ -88,4 +57,11 @@ func init() {
 		_ = fmt.Errorf("kvm option not supplied")
 	}
 
+	// options for vm generic commands related to HTTP Request
+	vmCmd.PersistentFlags().StringVar(&HttpIP, "http-ip", "127.0.0.1", "IP of non-k8s control plane")
+	vmCmd.PersistentFlags().StringVar(&HttpPort, "http-port", "8080", "IP and port of http request")
+	vmCmd.PersistentFlags().BoolVar(&IsNonK8sEnv, "non-k8s", false, "Enable if non-k8s environment/control-plane")
```

Comment:
It is better to keep this true by default since we will be using non-k8s mode more frequently. It is painful to type `--non-k8s` with every `karmor` command.
```suggestion
	vmCmd.PersistentFlags().BoolVar(&IsNonK8sEnv, "non-k8s", true, "Enable if non-k8s environment/control-plane")
```

Reply:
@nyrahul  Reason for default false is that, this utility is used by both k8s and non-k8s for policy enforcement. In the k8s case, for VM direct, the policy enforcement will  happen over GRPC. Hence thought not to disturb the existing flow. Also the use of non-k8s option is required only on policy enforcement and installation script download.